### ISSUE TITLE
[Feature Request] Add alt property to v-parallax for the img alt tag

### ISSUE THREAD

**Issue by leolux** [vuetifyjs/vuetify#8871]
### Problem to solve
SEO requires alt tags on images.

### Proposed solution
The alt tag of the `<img>` element should be set via v-parallax.

**Comment by redahas** [comment]
@leolux 
VParallax will accept an alt prop and set it on the img element. eg:
`<v-parallax src="https://cdn.vuetifyjs.com/images/backgrounds/vbanner.jpg" alt="Parallax image">`

However, if one is not provided then the img element is left with an empty alt attribute as currently the default alt value is an empty string. Are you asking for a default string to be put in place so that if one is not provided there is something there? Something like 'Parallax component image' or ... any ideas*?

**Comment by KaelWD** [comment]
`alt` is empty by default as v-parallax is often used as a background image for other content. 

> Omitting `alt` altogether indicates that the image is a key part of the content and no textual equivalent is available. Setting this attribute to an empty string `(alt="")` indicates that this image is not a key part of the content (it’s decoration or a tracking pixel)

If you want a specific alt text you can already supply it with the `alt` prop.
